packages/core/src/enqueue_links/shared.ts

```diff
@@ -126,17 +126,41 @@ export function constructRegExpObjectsFromRegExps(regexps: RegExpInput[]): RegEx
 /**
  * @ignore
  */
-export function createRequests(requestOptions: (string | RequestOptions)[], urlPatternObjects?: UrlPatternObject[]): Request[] {
+export function createRequests(
+    requestOptions: (string | RequestOptions)[],
+    urlPatternObjects?: UrlPatternObject[],
+    blacklistPatternObjects?: UrlPatternObject[],
```

Comment:
You can inline this with 
```suggestion
    blacklistPatternObjects: UrlPatternObject[] = [],
```